app/server.js

```diff
@@ -1,13 +1,205 @@
-var http = require('http');
-var fs = require('fs');
-var finalHandler = require('finalhandler');
-var queryString = require('querystring');
-var Router = require('router');
-var bodyParser   = require('body-parser');
-var uid = require('rand-token').uid;
+var http = require('http')
+var fs = require('fs')
```

Comment:
Stay consistent with variable declaration instead of switching between var and let/const